bin/internal/material_fonts.version

```diff
@@ -1 +1 @@
-https://storage.googleapis.com/flutter_infra/flutter/fonts/13ac995daa9dda0a6ba0a45f1fccc541e616a74c/fonts.zip
+${FLUTTER_STORAGE_BASE_URL}/flutter_infra/flutter/fonts/13ac995daa9dda0a6ba0a45f1fccc541e616a74c/fonts.zip
```

Comment:
it's a bit weird to have this replacement string here. Why not just make these files have relative paths and always prefix the storage url?

Reply:
I assumed it was done this way to make it easier to customize where the fonts (and now also gradle wrapper) were downloaded from, but I can make it a relative URL instead.